spec/lib/pdf_filler/f13614c_pdf_spec.rb

```diff
@@ -608,8 +648,8 @@
                 expect(intake_pdf.hash_for_pdf[additional_comments_key]).to eq(<<~COMMENT.strip)
                   Other income types: garden gnoming
                   Additional Dependents:
-                  (a) Polly Pony (b) 8/27/2018 (c) Baby (d) 5 (e) N (f) Y (g) S (h) N (i) Y CVP: ////
-                  (a) Patrick Pony (b) 3/11/2019 (c) Son (d) 8 (e) N (f) Y (g) S (h) N (i) N CVP: ////
+                  (a) Polly Pony (b) 8/27/2018 (c) Baby (d) 5 (e) S (f) N (g) Y (h) N (i) Y CVP: ////
```

Comment:
i think this will get overwritten in https://github.com/codeforamerica/vita-min/pull/5431/files#diff-55f9b31b5196260702be6515a3483ad05eca50729a5324079b6c49fcd71250df